packages/turbo-kovan-subgraph/src/mappings/AmmFactory.ts

```diff
@@ -85,7 +85,11 @@ export function handleLiquidityChangedEvent(event: LiquidityChanged): void {
   let senderId = event.params.user.toHexString();
   let liquidityEntity = getOrCreateLiquidity(id, true, false);
   getOrCreateMarket(marketId);
-  getOrCreateSender(senderId);
+  let sender = getOrCreateSender(senderId);
+  sender.totalLiquidity = sender.totalLiquidity ?
```

Comment:
@bconfortin Wherever you initialize sender just set the totalLiquidity to zero which will allow you to drop this if statement.